### ISSUE TITLE
Tonofilament clumping

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7253]
**Preferred term label:**
Tonofilament clumping

**Synonyms**
Clumping of the keratin tonofilaments

**Definition (free text, please give PubMed ID)**
Formation of lumps of tonofilaments, which are bundles of keratin filaments.  In some but not all epithelia, keratin filaments are conspicuously bundled as tonofilaments. Inside the cell they braid the nucleus, span through the cytoplasm and are attached to the cytoplasmic plaques of the typical epithelial cell-cell junctions, the desmosomes.

PMID:7529919
PMID:18461349
PMID:7529919

**Parent term (use hpo.jax.org/app)**
Abnormality of skin morphology

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

OMIM:619555   PMID:10730767  2/2

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
